packages/features/instant-meeting/handleInstantMeeting.ts

```diff
@@ -86,6 +89,69 @@ const handleInstantMeetingWebhookTrigger = async (args: {
   }
 };
 
+const triggerBrowserNotifications = async (args: {
+  title: string;
+  connectAndJoinUrl: string;
+  teamId: number;
+}) => {
+  const { title, connectAndJoinUrl, teamId } = args;
+
+  const subscribers = await prisma.membership.findMany({
+    where: {
+      teamId,
+      accepted: true,
+    },
+    select: {
+      user: {
+        select: {
+          id: true,
+          NotificationsSubscriptions: {
+            select: {
+              id: true,
+              subscription: true,
+            },
+          },
+        },
+      },
+    },
+  });
```

Comment:
NIT: Could we not do a trpc loggedInViewer here? Would probably be the better route.

Reply:
> Could we not do a trpc loggedInViewer here? 

Could you elaborate more on this?

For more context when someone submits a instant booking meeting handleInstantMeeting is executed (just like handleNewBooking for normal event types) and many other tasks are executed like triggering webhook, creating video call url. etc.